src/components/pages/ReportForm/Steps/Step1.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+.step1
+  h3 {{ $t('step1.title') }}
+
+  label {{ $t('step1.name') }}
+  input(name="name", 
+        v-model="owner.name",
+        type="text", 
+        v-validate="'required'", 
+        placeholder="Joe Doe")
+  span(v-show="errors.first('name')" class="help is-danger") {{ errors.first('name') }}
```

Comment:
The way to add classes in pug is writing the element this way:
```jade
span.help.is-danger(v-show='errors.first("name"))
  | {{ errors.first('name') }}
```
Note that the two classes are written outside the attributes parenthesis and one after the other, that way you can write a more readable code :smile: